assets/manifest_template.toml

```diff
@@ -1,5 +1,5 @@
 #
```

Comment:
blocking: I'm seeing profile run twice. Could be my dotfiles but I think you were seeing that too @limeytexan so maybe there actually is something wrong
```
> flox list -c
[profile]
common = """
  echo "common"
"""

[options]
systems = ["aarch64-darwin"]

> flox activate
✅ You are now using the environment in current directory.
To stop using this environment, type 'exit'

common
...
common
```

Reply:
Good spot - added test and fix in 40a00c4a33e15fb76ce70cdfb718d4a8e4e8056a, resolving conversation.

Reply:
By way of background I expected to see it twice because I was running flox activate in each of my .zshrc and .zshenv files; I hadn't noticed that it was happening elsewhere.